src/js/index.js

```diff
@@ -14,9 +14,9 @@
       const target = this.hash;
       const $target = $(target); // use hash as a selector to find the section by id
       $('html, body').stop().animate({
-        'scrollTop': $target.offset().top
+        'scrollTop': ($target.offset().top - 60) // scroll to section location minus the height of navbar
```

Comment:
You can future proof this by programmatically getting the height. `$('.navbar').height();`

May be extra, but you could even do a condition check if it's fixed. I'll leave it up to you!